configure.ac

```diff
@@ -3802,6 +3802,11 @@ AS_VAR_IF([have_uuid], [missing], [
   ])
 ])
 
+if test "$ac_sys_system" = "NetBSD"; then
```

Comment:
Please add a comment with a reference to `gh-124228` explaining that libuuid is available but doesn't fit Python requirements. I understand that NetBSD libuuid only supports UUID4.